controllers/managedocs_controller.go

```diff
@@ -662,8 +674,38 @@ func (r *ManagedOCSReconciler) reconcileStorageCluster() error {
 }
 
 func (r *ManagedOCSReconciler) getDesiredConvergedStorageCluster() (*ocsv1.StorageCluster, error) {
-	sizeAsString := r.addonParams[storageClassSizeKey]
+	sc, err := r.getDesiredStorageCluster(templates.StorageClusterTemplate.DeepCopy())
+	if err != nil {
+		return nil, err
+	}
+	return sc, nil
+}
```

Comment:
Why do we need another function that basically does what this function is supposed to do?
right now this function body is equivalent to:
```
       return r.getDesiredStorageCluster(templates.StorageClusterTemplate.DeepCopy())
```
which is meaningless 


Also, why are we touching converged in the first place?